app/presenters/sufia/select_type_presenter.rb

```diff
@@ -0,0 +1,35 @@
+module Sufia
+  class SelectTypePresenter
+    def initialize(concern)
+      @concern = concern
+    end
+
+    attr_reader :concern
+
+    def icon_class
+      translate('icon_class')
```

Comment:
We need to have a different icon for each worktype. See: https://cloud.githubusercontent.com/assets/101482/19989183/0ca5f6a4-a1e2-11e6-8f7f-44845724f209.png